### PR TITLE
Rename ONNX_PREVIEW_DOMAIN to AI_ONNX_PREVIEW_TRAINING_DOMAIN.

### PR DESCRIPTION
**Pull request by codemzs**

### CODE REVIEW

**Comment by linkerzhang** [comment]
https://github.com/onnx/onnx/blob/673aef09ba6cb56e7c34f807255f8372f253c0ed/onnx/common/constants.h#L14

**Comment by codemzs** [comment]
@linkerzhang thanks, I was searching for ONNX_PREVIEW_DOMAIN, but looks like we also have AI_ONNX_PREVIEW_DOMAIN

File: onnx/defs/schema.h [hunk 1]
```diff
@@ -988,7 +988,7 @@ OpSchema GetOpSchema();
 
 #define ONNX_PREVIEW_OPERATOR_SET_SCHEMA(name, ver, impl) \
```
Comment (linkerzhang) [inline onnx/defs/schema.h:989]: please also change this macro's name accordingly.

Comment (codemzs) [reply onnx/defs/schema.h:989]: ONNX_PREVIEW_TRAINING_OPERATOR_SET_SCHEMA look good?

**Pull request merged**